scripts/linters/linter_utils_test.py

```diff
@@ -170,3 +171,20 @@ def count_calls(a, b, c=0, d=1):
         count_calls(5, 6, d=3, c=2)
         self.assertEqual(call_counter[(5, 6, 3, 2)], 0)
         self.assertEqual(call_counter[(5, 6, 2, 3)], 1)
+
+
+class RedirectStoutTest(test_utils.GenericTestBase):
+    """Tests for the redirect_stdout function."""
+
+    def test_redirect_stdout(self):
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'r+') as tmp:
```

Comment:
Inline temp_file.name here. In general, if you are using a construct just once, then no need to declare a new variable, unless the name of that added variable brings additional clarity. Otherwise it just becomes yet another thing that the reader has to keep track of.

Also change tmp to tmp_file_contents or similar. There's too many "tmp" things floating around the place and I had to think a bit to figure out what was what. Some of this can be avoided by more precise naming.

Reply:
Done.